dartagnan/src/main/java/com/dat3m/dartagnan/configuration/Property.java

```diff
@@ -31,6 +32,7 @@ public String toString() {
             case DATARACEFREEDOM -> "Data-race freedom (SVCOMP only)";
             case TERMINATION -> "Termination";
             case CAT_SPEC -> "CAT specification";
+            case TRACKABILITY -> "Memory Trackability";
```

Comment:
Unless we see a use case of this outside SVCOMP, I would add "(SVCOMP only)" for consistency with data race freedom